.vscode/launch.json

```diff
@@ -7,11 +7,24 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Launch Program",
+      "name": "Launch HOPRd (localhost)",
       "skipFiles": ["<node_internals>/**"],
       "program": "${workspaceFolder}/packages/hoprd/src/index.ts",
-      "args": ["--admin", "--init", "--rest", "--password=''", "--data=/tmp/debug"],
-      "outFiles": ["${workspaceFolder}/**/*.js"]
+      "args": [
+        "--admin",
+        "--init",
+        "--rest",
+        "--password='d3f4uL+%'",
+        "--testNoAuthentication",
+        "--data=/tmp/debug",
+        "--identity=/tmp/debug-identity",
+        "--environment='hardhat-localhost'"
+      ],
+      "env": {
+        "DEBUG": "hopr*"
+      },
+      "console": "integratedTerminal",
+      "outFiles": ["${workspaceFolder}/packages/**/*.js"]
```

Comment:
Why don't write the files to `packages/**/lib/*.js` ?

Reply:
I wanted to see if I could catch some exceptions in our `node_modules` packages (e.g. the `libp2p` error ("PeerId is not a valid peer id")) but didn't work.

Will update it to suggestion.

Reply:
Solved in 805c404